tests/test_thespruceeats_2.py

```diff
@@ -0,0 +1,107 @@
+                        "1/4 teaspoon salt",
+                        "87 grams all-purpose flour",
+                        "2/3 cup peppermint candies",
+                    ],
+                    purpose="For the Brownies:",
+                ),
+            ],
+            self.harvester_class.ingredient_groups(),
+        )
+
+    def test_instructions(self):
+        expected_instructions = (
+            "Gather the ingredients.\n"
+            "Position a rack in the center of the oven and heat to 350 F. Grease an 8-inch square cake pan with softened butter. Line the bottom with a large sheet of parchment paper that extends up and over two opposite sides of the pan.\n"
+            "To make the peppermint swirl, place the cream cheese, confectioners' sugar, salt, egg, extract, and food coloring in a medium bowl and beat with a hand mixer on medium to medium high speed until combined and fluffy. You can also use a stand mixer.\n"
+            "To make the brownies, microwave the butter, cocoa powder, and sugar on high in a large microwave-safe bowl in a few 30-second bursts, whisking after each, until combined. The batter will be quite thick and fudgy-looking. If you do not have a microwave, you may place the ingredients in a heat-proof bowl over a pot of simmering water on the stovetop until combined.\n"
+            "Whisk in the extract. Gently whisk in the eggs one at a time.\n"
+            "Sprinkle the salt over the wet ingredients and whisk it in.\n"
+            "Finally, fold in the flour and 1/3 cup of the crushed candy until the last streak of flour disappears. Don’t overmix.\n"
+            "Scrape the batter into the prepared pan, reserving 3/4 cup of it.\n"
+            "Dollop the swirl mixture over the brownie batter in the pan—about 9 dollops, using a 1 1/2 tablespoon portion scoop, if you have it.\n"
+            "Dollop the reserved brownie batter in between the swirl dollops. Run a long wooden skewer through both the brownie batter and the pink cream cheese mixture, until the top is completely covered in decorative swirls. Don’t swirl too much: big swirls are what you want, as they are most dramatic. And don’t swirl too deep or the swirl can get lost in the batter.\n"
+            "Bake for 30 to 35 minutes, rotating the pan at the halfway point. At the 25 minute mark, sprinkle the remaining 1/3 cup of candy over the top of the brownies. The brownies are ready when a cake tester inserted into the brownie—not the swirl—comes out with a few moist crumbs—do not over bake.\n"
+            "Let cool to room temperature. Lift the brownies out of the pan by the parchment handles. Run a butter knife around the edges not covered by the paper, if there’s resistance. Slice into 9 equal sized squares and serve room-temperature, or refrigerate for an hour and serve chilled (my preference)."
+        )
+        self.assertEqual(
+            expected_instructions.strip(), self.harvester_class.instructions().strip()
```

Comment:
Does this expectation fail if we remove the `.strip()` method calls from both values?

Reply:
My mistake! Been avoiding the .strip in favor of normalize_string, I'll revisit later today.